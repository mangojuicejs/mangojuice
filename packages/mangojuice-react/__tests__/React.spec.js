import AbstractTests from 'mangojuice-react-core/tests';
import React from 'react';
import ReactDOM from 'react-dom';
import Subscribe from 'mangojuice-react';

AbstractTests(React, ReactDOM, Subscribe, 'React');
