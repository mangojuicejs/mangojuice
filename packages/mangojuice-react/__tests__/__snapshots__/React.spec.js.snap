// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React interface view renderer: React shuold render a stateless view of a block 1`] = `
Array [
  Object {
    "a": "test",
    "nested": null,
  },
  Array [
    Object {
      "a": "another",
    },
  ],
  Object {
    "a": "another",
  },
]
`;
